compiler/rustc_session/src/options.rs

```diff
@@ -1184,6 +1194,8 @@ options! {
         "pass `-install_name @rpath/...` to the macOS linker (default: no)"),
     panic_abort_tests: bool = (false, parse_bool, [TRACKED],
         "support compiling tests with panic=abort (default: no)"),
+    panic_in_drop: PanicStrategy = (PanicStrategy::Unwind, parse_panic_strategy, [TRACKED],
+        "panic strategy for panics in drops"),
```

Comment:
Add accepted values list and default (like in some other options), plus maybe unstable book entry?